app/helpers/date-from-now.ts

```diff
@@ -1,21 +1,25 @@
-import { helper } from '@ember/component/helper';
 import { formatDistanceStrictWithOptions } from 'date-fns/fp';
+import { inject as service } from '@ember/service';
+import Helper from '@ember/component/helper';
+import TimeService from 'codecrafters-frontend/services/time'
 
-const dateFromNow = helper(function dateFromNow(
-  [date]: [(Date | null | undefined)?],
-  { currentDate = new Date() }: { currentDate?: Date | undefined } = {},
-): string {
-  if (date === undefined || date === null) {
-    return '';
+interface DateFromNowSignature {
+  Args: {
+    Positional: Array<Date | null | undefined>;
   }
+}
 
-  return formatDistanceStrictWithOptions({ addSuffix: true }, currentDate, date);
-});
+export default class DateFromNow extends Helper<DateFromNowSignature> {
+  @service declare time: TimeService;
+
+  compute(positional: [Date]) {
```

Comment:
```suggestion
  compute(positional: DateFromNowSignature['Args']['Positional']) {
```

Note that we need to handle null / undefined values here, don't think that's done at the moment